libraries/AP_Avoidance/AP_Avoidance.h

```diff
@@ -189,6 +189,7 @@ class AP_Avoidance {
     AP_Int8     _fail_time_horizon;
     AP_Int16    _fail_distance_xy;
     AP_Int16    _fail_distance_z;
+    AP_Int32    _fail_altitude_minimum;
```

Comment:
If the expected value is in the 0-5 range, and a crazy huge number is 100, mind if we change this to at least 16bit? 32 is sill, even for big-boy full size aircraft.